app/views/timeline/entry.scala.html

```diff
@@ -46,13 +46,11 @@
 }
 case GameEnd(playerId, opponent, win, perfKey) => {
 @lila.rating.PerfType(perfKey).map { perf =>
-<a href="@routes.Round.player(playerId)" data-icon="@perf.iconChar" class="glpt">
-  @win match {
-  case Some(true) => {Victory}
-  case Some(false) => {Defeat}
-  case _ => {Draw}
-  }</a> vs @userIdLink(opponent, withOnline = false)<br />
-in @perf.name
+@trans.xVsYinZ(Html("""<a href="%s" data-icon="%s" class="glpt"> %s</a>""".format(routes.Round.player(playerId), perf.iconChar, win match {
+  case Some(true) => trans.gameVictory()
+  case Some(false) => trans.defeat()
+  case _ => trans.draw()
+})), userIdLink(opponent, withOnline = false), perf.name)
```

Comment:
should we maybe have full translatable sentences to avoid "translation puzzles"?

`Victory vs %s$1 in %s$2`
`Defeat vs %s$1 in %s$2`
`Draw vs %s$1 in %s$2`